templates/daemon.sysv.erb

```diff
@@ -1,32 +1,32 @@
 #!/bin/bash
 #
-#       /etc/rc.d/init.d/node_exporter
+#       /etc/rc.d/init.d/<%= @name %>
 #
-#       Daemonize the prometheus node exporter
+#       Daemonize the prometheus <%= @name %>
 #
 # chkconfig:   2345 95 20
-# description: Prometheus exporter for machine metrics, written in Go with pluggable metric collectors.
-# processname: node_exporter
-# pidfile: /var/run/node_exporter/pidfile
+# description: Prometheus Exporter <%= @name %>
+# processname: <%= @name %>
+# pidfile: /var/run/<%= @name %>/pidfile
 
 # Source function library.
 . /etc/init.d/functions
 
-DAEMON=<%= scope.lookupvar('prometheus::node_exporter::bin_dir') %>/node_exporter
-PID_FILE=/var/run/node_exporter/node_exporter.pid
-LOG_FILE=/var/log/node_exporter
+DAEMON=<%= @bin_dir %>/<%= @name %>
+PID_FILE=/var/run/<%= @name %>/<%= @name %>.pid
+LOG_FILE=/var/log/<%= @name %>
 
-[ -e /etc/sysconfig/node_exporter ] && . /etc/sysconfig/node_exporter
+[ -e /etc/sysconfig/<%= @name %> ] && . /etc/sysconfig/<%= @name %>
```

Comment:
I think `/etc/sysconfig` is a RedHat thing? This should read from `/etc/default` for Debian systems.